### ISSUE TITLE
Smart Spaces filters should work as "any"

### ISSUE THREAD

**Issue by abraverm** [Make-md/makemd#145]
When add more than one matching filter, the joined filters should be treated as "any".
For example:
- Match any  of the following
  - parent is folder_1
- Match any of the following
  - parent is folder_2

The result should be all the notes from both folders.

Currently it shows no notes (applying "all").

Another option is to grant the user to decide if it should be "any" or "all".

**Comment by jp-cen** [comment]
For now, you can just add both conditions under the first group since you can have multiple conditions under the same group and it will work based on the group’s filter type

**Comment by abraverm** [comment]
Right, but I want to filter the second group with additional conditions (tags, properties, etc).

**Comment by hirowa** [comment]
I have the same problem if I try to filter a "multi option" property by "is any of" the filters are ignored, it only works if it is "is none of". I had no problem doing that, but when I added a new variable to that property I need to go an change the filter again.
